benchmarks/ImageProcessing/BuddyCorr2DBenchmark.cpp

```diff
@@ -50,6 +57,12 @@ intptr_t sizesInputBuddyCorr2D[2];
 intptr_t sizesKernelBuddyCorr2D[2];
 intptr_t sizesOutputBuddyCorr2D[2];
 
+// Declare Boundary Options supported.
+enum BoundaryOption {constant_padding, replicate_padding};
+
+// Define Boundary option selected.
+int BoundaryType;
```

Comment:
Why do we need to define this variable as `int`? 
Please try using a `bool` instead, since it self-documents its utility as a flag and would improve overall readability imo.

Reply:
I thought in future maybe there will be support of more extrapolation methods like `BORDER_REFLECT`, `BORDER_WRAP` etc. But for now I guess boolean flag would suffice so I have updated it.

Reply:
Please use `BoundaryOption` as the type for variable `BoundaryType`.